### DISCUSSION TITLE
telegram bot for bet365

### DISCUSSION THREAD

**Discussion by DouglassPG** [community/community#32471] [Code Security]
Good night, I would like help I am inexperienced in the field of programming, I would like to create or if it is already ready a bot for the telegram of annelise from bet365.
example:
- one of the settings that I use to enter a bet is the "corner limit", what are the game settings so that it can be good for me to enter this strategy:
Shot on goal - 4
Side kick - 6
These two have to be at least 8 together.
Ball possession - 55
Dangerous attack - = 1 or >1 (in case the dangerous attack is divided by the game time, the bot being in the pre-defined settings it sends the game link in the group

**Comment by Krakensaller** [comment]
> Good night, I would like help I am inexperienced in the field of programming, I would like to create or if it is already ready a bot for the telegram of annelise from bet365.
> example:
> - one of the settings that I use to enter a bet is the "corner limit", what are the game settings so that it can be good for me to enter this strategy:
> Shot on goal - 4
> Side kick - 6
> These two have to be at least 8 together.
> Ball possession - 55
> Dangerous attack - = 1 or >1 (in case the dangerous attack is divided by the game time, the bot being in the pre-defined settings it sends the game link in the group
